app/_locales/pt_BR/messages.json

```
@@ -3,105 +3,180 @@
+    "message": "Versão, central de suporte e informações de contato"
   },
   "acceleratingATransaction": {
-    "message": "* Acelerar uma transação usando um preço de gás mais alto aumenta suas chances de a rede processá-la de forma mais rápida, mas isso nem sempre é garantido."
+    "message": "* Acelerar uma transação usando um Gas Price mais alto aumenta as chances de agilizar o processamento pela rede, mas nem sempre é garantido."
+  },
+  "acceptTermsOfUse": {
+    "message": "Eu li e concordo com $1",
+    "description": "$1 is the `terms` message"
+  },
+  "accessAndSpendNotice": {
+    "message": "$1 pode acessar e gastar até esse valor máximo",
+    "description": "$1 is the url of the site requesting ability to spend"
   },
   "accessingYourCamera": {
-    "message": "Acessando a sua câmera..."
+    "message": "Acessando sua câmera..."
   },
   "account": {
     "message": "Conta"
   },
   "accountDetails": {
-    "message": "Detalhes da Conta"
+    "message": "Detalhes da conta"
   },
   "accountName": {
-    "message": "Nome da Conta"
+    "message": "Nome da conta"
   },
   "accountOptions": {
-    "message": "Opções da Conta"
+    "message": "Opções da conta"
   },
   "accountSelectionRequired": {
     "message": "Você precisa selecionar uma conta!"
   },
+  "active": {
+    "message": "Ativo"
+  },
+  "activity": {
+    "message": "Atividade"
+  },
   "activityLog": {
-    "message": "registro de atividade"
+    "message": "log de atividades"
+  },
+  "addAccount": {
+    "message": "Adicionar uma conta"
   },
   "addAcquiredTokens": {
-    "message": "Adicione os tokens que você adquiriu usando o MetaMask"
+    "message": "Adicionar os tokens que você adquiriu usando o MetaMask"
   },
   "addAlias": {
-    "message": "Adicionar pseudônimo"
+    "message": "Adicionar alias"
   },
   "addNetwork": {
-    "message": "Adicionar Rede"
+    "message": "Adicionar rede"
   },
   "addRecipient": {
-    "message": "Adicionar Destinatário"
+    "message": "Adicionar destinatário"
   },
   "addSuggestedTokens": {
-    "message": "Adicionar Tokens Sugeridos"
+    "message": "Adicionar tokens sugeridos"
   },
   "addToAddressBook": {
-    "message": "Adicionar à lista de endereços"
+    "message": "Adicionar ao catálogo de endereço"
   },
   "addToAddressBookModalPlaceholder": {
-    "message": "Exemplo: John D."
+    "message": "por ex., John D."
   },
   "addToken": {
     "message": "Adicionar token"
   },
   "addTokens": {
-    "message": "Adicionar Tokens"
+    "message": "Adicionar tokens"
   },
   "advanced": {
-    "message": "Avançado"
+    "message": "Avançadas"
   },
   "advancedOptions": {
     "message": "Opções avançadas"
   },
   "advancedSettingsDescription": {
-    "message": "Acesse recursos do desenvolvedor, baixe registros de estado, redefina a conta, configure testnets e personalize RPC"
+    "message": "Acessar recursos do desenvolvedor, baixar logs de estado, redefinir conta, configurar testnets e RPC personalizada"
```

This sounds a bit "google-translated-ish". I can't really explain why. But here are a few changes I'd make:

1. Change `Acessar recursos DO desenvolvedor` with `Acessar recursos DE desenvolvedor`.
2. Change `Baixar logs de estado` with `Baixar logs`
3. Change `Configurar testnets e RPC personalizada` with `Configurar testnets e RPC personalizadaS`

---

Thanks! Does this look OK?

```suggestion
    "message": "Acessar recursos de desenvolvedor, baixar logs, redefinir conta, configurar testnets e RPC personalizadas"
```

---

Yes. It reads better :)